pages/transfer.vue

```diff
@@ -18,7 +18,7 @@
       <b-field>
         <Auth />
       </b-field>
-      <div v-if="$route.query.target" class="box--target-info">
+      <div v-if="$route.query.target && !isBSX" class="box--target-info">
```

Comment:
this was already fixed somehow smarter or?

Reply:
> smol

added discussed changes, thanks for patience and review